db/migrate/20240624115536_add_indices_to_invoices.rb

```diff
@@ -0,0 +1,10 @@
+class AddIndicesToInvoices < ActiveRecord::Migration[7.0]
+  def change
+    add_index :invoices, :currency
+    add_index :invoices, :issuing_date
+    add_index :invoices, :invoice_type
+    add_index :invoices, :status
+    add_index :invoices, :payment_status
```

Comment:
Indexes on `*status` should be avoided I think, the cardinality is very low so it is not really efficient but it could make senses to combine them with an ID field lie `organization_id` or `customer_id` depending on context.